tests/exec-tests.sh

```diff
@@ -14,11 +14,15 @@ ADD tests/exec-steps.sh tests/step-* /var/lib/tests/
 
 # Do not use the terminal in tests
 RUN sed -i 's|</dev/tty||g' /var/lib/githooks/install.sh && \\
+    sed -i -E 's|ACCEPT_CHANGES=.*|ACCEPT_CHANGES=\${ACCEPT_CHANGES}|' /var/lib/githooks/install.sh && \\
```

Comment:
I tested this on your branch, we do not need to change the install script with these 3 lines, please remove them.